### PR TITLE
Only send an email for the comments added when CI fails

### PR DESCRIPTION
**Pull request by pypingou**
Relates to https://github.com/fedora-infra/bodhi/issues/3431

Signed-off-by: Pierre-Yves Chibon <pingou@pingoured.fr>

### CODE REVIEW

File: bodhi/server/models.py [hunk 1]
```diff
@@ -3706,6 +3708,9 @@ def comment_on_test_gating_status_change(target, value, old, initiator):
         """
         Place comment on the update when ``test_gating_status`` changes.
 
+        Only notify the users by email if the new status is not in ``waiting``,
```
Comment (abompard) [inline bodhi/server/models.py]: This looks like it's saying the opposite from what it's doing. Shouldn't `not` be removed? Also, what about the `waiting` state, it's not in the code below.

File: bodhi/tests/server/test_models.py [hunk 2]
```diff
@@ -4164,6 +4165,37 @@ def test_comment_on_test_gating_status_change(self):
...
+        self.assertEqual(len(self.obj.comments), 1)
+
+        # Let's set test_gating_status to 'waiting' once again.
+        self.obj.test_gating_status = TestGatingStatus.waiting
+
+        # Check that still only one email was sent:
+        self.assertEqual(mail.send.call_count, 1)
+
+        # We should have still only one comment about test_gating_status change.
+        self.assertEqual(len(self.obj.comments), 2)
```
Comment (abompard) [inline bodhi/tests/server/test_models.py:4197]: I don't understand, you want to test that there's still only one comment, but you assert that the length is 2 here.

File: bodhi/server/models.py (hunk 1 above)
Comment (pypingou) [reply bodhi/server/models.py]: sorry missed to update the comment in both sections, will adjust

**Comment by abompard** [comment]
Obsoleted by #3435

**Pull request closed without merging**